scopes/component/status/status-cmd.ts

```diff
@@ -220,47 +220,50 @@ or use "bit merge [component-id] --abort" for previous "bit merge" command)\n`;
     const autoTagPendingOutput = immutableUnshift(
       autoTagPendingComponents.map((c) => format(c)),
       autoTagPendingComponents.length
-        ? chalk.underline.white('components pending to be tagged automatically (when their dependencies are tagged)')
+        ? chalk.underline.white('components pending auto-tag (when their modified dependencies are tagged)')
         : ''
     ).join('\n');
 
-    const invalidDesc = '\nthese components were failed to load.\n';
+    const invalidDesc = '\nthese components failed to load.\n';
     const invalidComponentOutput = immutableUnshift(
       invalidComponents.map((c) => format(c.id, false, getInvalidComponentLabel(c.error))).sort(),
       invalidComponents.length ? chalk.underline.white(statusInvalidComponentsMsg) + invalidDesc : ''
     ).join('\n');
 
-    const locallySoftRemovedDesc = '\n(tag/snap and export them to update the remote. to undo, run "bit recover")\n';
+    const locallySoftRemovedDesc =
+      '\n(tag/snap and export the components to update the deletion to the remote. to undo deletion, run "bit recover")\n';
     const locallySoftRemovedOutput = immutableUnshift(
       locallySoftRemoved.map((c) => format(c)).sort(),
       locallySoftRemoved.length ? chalk.underline.white('soft-removed components locally') + locallySoftRemovedDesc : ''
     ).join('\n');
 
     const remotelySoftRemovedDesc =
       '\n(use "bit remove" to remove them from the workspace. use "bit recover" to undo the soft-remove)\n';
+    // @david shouldn't we auto-remove them locally when they've been removed on the remote?
```

Comment:
we actually do that.
when you soft-remove, we also remove locally.
when you merge a lane with soft-removed and you have the component locally, we remove it for you. 
same when you `checkout head` and the head has this component as removed.

However, you can choose to "bit import' it for whatever reason, and in that case we want to tell you that it is removed on the remote.

Reply:
So this output is incorrect then and should be removed?

Reply:
No, the output is good. It happens not when you currently soft-removed. It happens when a component was soft-removed in the past, and it's in your workspace. (e.g. with bit-import).

Reply:
ok, so i'm going to re-suggest updating this description and open a new thread on this

Reply:
actually this looks fine now, think i missed that there was a message about soft-removed from the remote